cockatrice/src/main.cpp

```diff
@@ -91,8 +85,9 @@ int main(int argc, char *argv[])
 {
     QApplication app(argc, argv);
 
+    qInstallMessageHandler(CockatriceLogger);
     if (app.arguments().contains("--debug-output"))
```

Comment:
This was my exact question when i started thinking about this PR. Most users are running Windows, and getting logs from there was quite impossible.
Please note that logs are always collected, the `--debug-output` option only enables writing the log to a file named `qdebug.txt` to mimic the old behavior.

Since Cockatrice is a UI app, now that the log is available from inside the UI i don't think that writing it to a file is so important, but i suppose that it can still be useful for automated testing purposes (run, collect logs, grep for errors).